spec/acceptance/init_spec.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require 'spec_helper_acceptance'
+
+describe 'anubis' do
+  describe 'with defaults' do
+    it_behaves_like 'an idempotent resource' do
+      let(:manifest) do
+        <<-PUPPET
+        include anubis
+        PUPPET
+      end
```

Comment:
For reference, you can shorten this if you want (but don't consider it blocking):
```suggestion
      let(:manifest) { 'include anubis' }
```